src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable prettier/prettier */
+import React, { useState } from 'react';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todos: Todo[];
+  filterStatus: (status: string) => void;
+}
+
+export const Footer: React.FC<Props> = ({ todos, filterStatus }) => {
+  const [statusFilter, setStatusFilter] = useState<
+  'all' | 'active' | 'completed'
+  >('all');
```

Comment:
you don't need this state here, just pass the statusFilter from app.tsx

Reply:
i don't know what's wrong but in my local server all be good 
![image](https://github.com/user-attachments/assets/84e5bada-a60d-4d12-b75b-46fd4c4c98b6)

Reply:
i fix this in app.tsx line 86